### PR TITLE
Arxiv malformed

### PR DESCRIPTION
**Pull request by brucemiller**
A few more easy bits...

### CODE REVIEW

File: lib/LaTeXML/Core/Document.pm [hunk 1]
```diff
@@ -884,7 +884,7 @@ sub closeToNode {
 
 # This closes all nodes until $node is closed.
 sub closeNode {
-  my ($self, $node) = @_;
+  my ($self, $node, $ifopen) = @_;
```
Comment (dginev) [inline lib/LaTeXML/Core/Document.pm:887]: so this is providing for a `maybeCloseNode` variant where `$ifopen` is the `$maybe_flag` ? :> Neat!

**Review by dginev — APPROVED** [review]
Nice, these look great!

File: lib/LaTeXML/Package/TeX.pool.ltxml [hunk 2]
```diff
@@ -1991,7 +2004,7 @@ DefConstructor('\hbox BoxSpecification HBoxContents', sub {
     if ($issvg) {    # ODDLY, svg:g isnt necessarily balanced in tikz?
       $document->maybeCloseElement('svg:g'); }
     else {
-      $document->closeNode($node); }
+      $document->closeNode($node, 1); }
```
Comment (dginev) [inline lib/LaTeXML/Package/TeX.pool.ltxml:2007]: but yes, returning here a few months from today, I think both of us would enjoy more seeing:
```perl
       $document->maybeCloseNode($node); }
```

as a clearer signal of what we intended to develop.

Comment (brucemiller) [reply lib/LaTeXML/Package/TeX.pool.ltxml]: good idea; thanks!

**Comment by dginev** [comment]
Thanks! Feel free to merge here if you're done -- I would like to rebase on your branch, and make sure I didn't end up with any conflicts. My PR is close to complete as well - just a handful papers more...

**Pull request merged**